src/annotator/plugin/bucket-bar.coffee

```diff
@@ -20,7 +20,7 @@ scrollToClosest = (anchors, direction) ->
       return acc
 
     {start, next} = acc
-    rect = highlighter.getBoundingClientRect(anchor.highlights)
+    rect = anchor.range.getBoundingClientRect()
```

Comment:
`dom-wrap-highlighter/index.coffee` has a note above the `getBoundingClientRect` documentation that explains why `Range.getBoundingClientRect` was not used, with a reference to this unresolved Chrome bug: https://bugs.chromium.org/p/chromium/issues/detail?id=324437

My guess is that we probably want to continue using our own implementation of this method, but move the function + tests to a separate module.